### ISSUE TITLE
Cannot find module '/app/src/Utils/discordapi'

### ISSUE THREAD

**Issue by namekinggamerboy** [Ankrad/Discord-Bot-List#25]
show me error :-
```
[2020-01-25 02:15:39] Uncaught Promise Error: 
[2020-01-25 02:15:39] Error: Cannot find module '/app/src/Utils/discordapi'
[2020-01-25 02:15:39] Require stack:
[2020-01-25 02:15:39] - /app/src/Routes/api/bots/submit.js
[2020-01-25 02:15:39] - /app/src/Routes/api/bots.js
[2020-01-25 02:15:39] - /app/src/Routes/api/index.js
[2020-01-25 02:15:39] - /app/src/Routes/index.js
[2020-01-25 02:15:39] - /app/src/Structures/App.js
[2020-01-25 02:15:39] - /app/src/index.js
[2020-01-25 02:15:39]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:610:15)
[2020-01-25 02:15:39]     at Function.Module._resolveFilename (/rbd/pnpm-volume/1ee719b1-fad5-4465-87ac-a51598473aef/node_modules/.registry.npmjs.org/module-alias/2.2.2/node_modules/module-alias/index.js:49:29)
[2020-01-25 02:15:39]     at Function.Module._load (internal/modules/cjs/loader.js:526:27)
[2020-01-25 02:15:39]     at Module.require (internal/modules/cjs/loader.js:666:19)
[2020-01-25 02:15:39]     at require (internal/modules/cjs/helpers.js:16:16)
[2020-01-25 02:15:39]     at Object.<anonymous> (/app/src/Routes/api/bots/submit.js:4:29)
[2020-01-25 02:15:39]     at Module._compile (internal/modules/cjs/loader.js:759:30)
[2020-01-25 02:15:39]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
[2020-01-25 02:15:39]     at Module.load (internal/modules/cjs/loader.js:628:32)
[2020-01-25 02:15:39]     at Function.Module._load (internal/modules/cjs/loader.js:555:12)```

**Comment by YasmijnVels** [comment]
try replace this in this file `/app/src/Routes/api/bots/submit.js`

`try {
        let r = req.app.get('client').guilds.get(GUILD_ID).roles.find(r => r.id === BOT_VERIFIERS_ROLE_ID);
        await req.app.get('client').guilds.get(GUILD_ID).channels.find(c => c.id === MOD_LOG_ID).send(`<@${newBot.owners[0]}> added <@${newBot.id}>: ${r}\n${process.env.DOMAIN}/bots/${bot.id}`);
        r.setMentionable(false);
        res.json({"redirect": "/success"});
    } catch (e) { console.error(e) }
});`

**Comment by Ankrad** [comment]
> show me error :-
> 
> ```
> [2020-01-25 02:15:39] Uncaught Promise Error: 
> [2020-01-25 02:15:39] Error: Cannot find module '/app/src/Utils/discordapi'
> [2020-01-25 02:15:39] Require stack:
> [2020-01-25 02:15:39] - /app/src/Routes/api/bots/submit.js
> [2020-01-25 02:15:39] - /app/src/Routes/api/bots.js
> [2020-01-25 02:15:39] - /app/src/Routes/api/index.js
> [2020-01-25 02:15:39] - /app/src/Routes/index.js
> [2020-01-25 02:15:39] - /app/src/Structures/App.js
> [2020-01-25 02:15:39] - /app/src/index.js
> [2020-01-25 02:15:39]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:610:15)
> [2020-01-25 02:15:39]     at Function.Module._resolveFilename (/rbd/pnpm-volume/1ee719b1-fad5-4465-87ac-a51598473aef/node_modules/.registry.npmjs.org/module-alias/2.2.2/node_modules/module-alias/index.js:49:29)
> [2020-01-25 02:15:39]     at Function.Module._load (internal/modules/cjs/loader.js:526:27)
> [2020-01-25 02:15:39]     at Module.require (internal/modules/cjs/loader.js:666:19)
> [2020-01-25 02:15:39]     at require (internal/modules/cjs/helpers.js:16:16)
> [2020-01-25 02:15:39]     at Object.<anonymous> (/app/src/Routes/api/bots/submit.js:4:29)
> [2020-01-25 02:15:39]     at Module._compile (internal/modules/cjs/loader.js:759:30)
> [2020-01-25 02:15:39]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
> [2020-01-25 02:15:39]     at Module.load (internal/modules/cjs/loader.js:628:32)
> [2020-01-25 02:15:39]     at Function.Module._load (internal/modules/cjs/loader.js:555:12)```
> ```

Are you sure you have all the files.

Firstly, that's only one error, not many. Also, it says it can't find the file `src/Utils/discordapi.js`, so make sure you have that file.

**Comment by namekinggamerboy** [comment]
Yes but flie true but find error😏

**Comment by Ankrad** [comment]
> Yes but flie true but find error😏

Umm I'm not sure then. Where are you hosting this?

**Comment by namekinggamerboy** [comment]
Glitch ☹️am use only mobile not pc

**Comment by Ankrad** [comment]
I haven't tried hosting it on glitch, but I'll give it a go and let you know.
Also, it'll be quite hard setting it up on mobile. gl!

**Comment by Ankrad** [comment]
@namekinggamerboy Actually, just realised, it's a typo. I'll fix and push the update. It's just replacing `discordapi` with `discordApi`.

**Comment by Ankrad** [comment]
Fixed: 5a89739
